topology/Langston University/LUCCRE/LUCILLE_downtime.yaml

```diff
@@ -9,17 +9,6 @@
   Severity: Outage
   StartTime: Jun 14, 2016 11:00 +0000
 # ----------------------------------------------------------
-- Class: SCHEDULED
```

Comment:
The SE entries in this file will need to be restored and instead you'll want to remove the squid entries